src/const/enumTypes.ts

```diff
@@ -33,6 +33,7 @@ export const authType = {
   public: 'public',
   serviceToService: 'service-to-service',
   userToService: 'user-to-service',
+  token: 'token',
```

Comment:
Because of this change, we'll need to add a migration. The former "userToService" Api configuration are now "token" Api configs.